src/main/java/com/github/manolo8/darkbot/gui/plugins/PluginUpdateHeader.java

```diff
@@ -0,0 +1,107 @@
+package com.github.manolo8.darkbot.gui.plugins;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.extensions.plugins.PluginHandler;
+import com.github.manolo8.darkbot.extensions.plugins.PluginUpdater;
+import com.github.manolo8.darkbot.gui.components.MainButton;
+import com.github.manolo8.darkbot.utils.I18n;
+import net.miginfocom.swing.MigLayout;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+public class PluginUpdateHeader extends JPanel {
+
+    private static final SimpleDateFormat dateFormatter = new SimpleDateFormat("EEE, d MMM, hh:mm a", Locale.ROOT);
```

Comment:
```suggestion
    private static final SimpleDateFormat dateFormatter = new SimpleDateFormat("EEE, d MMM, hh:mm a");
```
Do not force locale upon users, see the other comment tho cause this will need to be swapped by a java8 DateTimeFormatter